cmd/config.go

```diff
@@ -405,6 +423,26 @@ func getAddonManifests(reader *bufio.Reader) ([]string, error) {
 	return addonSlice, nil
 }
 
+func generateK8sVersionList(all bool) error {
+	if !all {
+		fmt.Println(metadata.DefaultK8sVersion)
+		return nil
+	}
+
+	allVersions := []string{}
+	for _, version := range metadata.K8sVersionsCurrent {
+		if _, ok := metadata.K8sBadVersions[version]; !ok {
+			allVersions = append(allVersions, version)
```

Comment:
why not just print the version instead of build a slice then iterating it?

Reply:
yup good point.